synapse/lib/aha.py

```diff
@@ -1221,8 +1230,8 @@ async def signHostCsr(self, csrtext, signas=None, sans=None):
 
         hostname = xcsr.subject.get_attributes_for_oid(c_x509.NameOID.COMMON_NAME)[0].value
 
-        hostpath = s_common.genpath(self.dirn, 'certs', 'hosts', f'{hostname}.crt')
-        if os.path.isfile(hostpath):
+        hostpath = self.certdir.getHostCertPath(hostname)
+        if hostpath:
```

Comment:
Should these be `is not None` checks?

Reply:
We could go either way; we're either getting a `None` or a `str` with some text in it. changed them to be inline with some of the other certdir checks in aha.py.